cmd/login/wc.go

```diff
@@ -282,18 +282,21 @@ func (r *runner) storeWCClientCertCredentials(c credentialConfig) (string, bool,
 	k8sConfigAccess := r.commonConfig.GetConfigAccess()
 	// Store client certificate credential either into the current kubeconfig or a self-contained file if a path is given.
 	if r.loginOptions.selfContainedClientCert && c.filePath != "" {
-		return printWCClientCertCredentials(k8sConfigAccess, r.fs, c)
+		return printWCClientCertCredentials(k8sConfigAccess, r.fs, c, r.loginOptions.contextOverride)
 	}
-	return storeWCClientCertCredentials(k8sConfigAccess, r.fs, c)
+	return storeWCClientCertCredentials(k8sConfigAccess, r.fs, c, r.loginOptions.contextOverride)
 }
 
-func getWCBasePath(k8sConfigAccess clientcmd.ConfigAccess, provider string) (string, error) {
+func getWCBasePath(k8sConfigAccess clientcmd.ConfigAccess, provider string, currentContext string) (string, error) {
 	config, err := k8sConfigAccess.GetStartingConfig()
```

Comment:
Added current context as an argument instead of getting it from the config to ensure that context override is applied correctly now that it is not stored in the kubeconfig file